### PR TITLE
Add basic input component

### PR DESCRIPTION
**Pull request by jpzwarte**

### CODE REVIEW

File: packages/core/src/input/input.stories.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <span slot="suffix">suffix</span>
+    </sl-input>
+  `
+};
+
+export const MinMaxLength: StoryObj = {
+  render: () => html`<sl-input maxlength="5" minlength="3" placeholder="Min 3, max 5 chars"></sl-input>`
+};
+
+export const Required: StoryObj = {
```
Comment (anna-lach) [inline packages/core/src/input/input.stories.ts:50]: Maybe example with custom validation message will be helpful?

File: packages/core/src/form.stories.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        display: flex;
+        flex-direction: column;
+      }
+      label {
+        margin-bottom: 0.25rem;
+      }
+      label:not(:first-of-type) {
+        margin-top: 1rem;
+      }
+      label:has(+ [required])::after {
```
Comment (anna-lach) [inline packages/core/src/form.stories.ts:95]: it looks like it's not working in Firefox :(

[the remark this answers is not in the document]
Comment (jpzwarte) [reply packages/core/src/form.stories.ts:95]: Yeah, FF doesn't support `:has` yet. But this is not production ready yet. So we have some time :)

File: packages/core/src/input/input.stories.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  render: () => html`<sl-input placeholder="I am required" required></sl-input>`
```
[the remark this answers is not in the document]
Comment (jpzwarte) [reply packages/core/src/input/input.stories.ts:51]: Right, something to look at.

**Pull request merged**